recipe/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: tensorflow-hub
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tensorflow/hub.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  noarch: python
+  
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bazel {{ bazel }}
+  host:
+    - git >=2.2
+    - python {{ python }}
+  run:
+    - python {{ python }}
+    - numpy {{ numpy }}
+    - six {{ six }}
+    - protobuf {{ protobuf }}
+    - tensorflow-base {{ tensorflow }}
+
+about:
+  home: https://www.tensorflow.org/hub/
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
Does this file need to be added?

Reply:
I see a LICENSE file in the root directory. @jayfurmanek - Are we adding Apache 2.0 License file for all the recipes?